wannierberri/__evaluate.py

```diff
@@ -203,7 +210,14 @@ def paralfunc(Kpoint,_system,_grid,npar_k,fftlib):
         
         try:
             if file_Klist is not None:
-                pickle.dump(K_list,open(file_Klist,"wb"))
+                
+                nk = len(K_list)
+                fw =  open(file_Klist,"wb")
+                nkpart = nk//10
+                for ink in range(nkpart):
+                    pickle.dump(K_list[10*ink:10*(ink+1)],fw)
+                if nk%10 !=0:
+                    pickle.dump(K_list[10*nkpart:],fw)
```

Comment:
it can be done in one `for` loop, e.g. 
```
for i in range(0,nk,10):
    pickle.dump(K_list[i:i+10],fw)
```

Reply:
I don't think so, nk%10 is not always 0.

Reply:
Not a problem 
![Screenshot from 2021-09-14 20-49-37](https://user-images.githubusercontent.com/22172977/133316644-e48cd661-a13b-4f23-9c72-cbffef6417e4.png)